code/notificationbuilder/src/main/res/layout/push_template_timer_expanded.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/timer_container_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation = "vertical"
+    android:theme="@style/DayNightTheme">
+
+    <RelativeLayout
+        android:id="@+id/relative_layout_timer"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_toLeftOf="@id/timer_text">
+
+            <TextView
+                android:id="@+id/notification_title"
+                style="@style/Title"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_alignParentLeft="true"
+                android:layout_alignParentStart="true" />
+
+            <TextView
+                android:id="@+id/notification_body_expanded"
+                style="@style/Body"
+                android:layout_width="wrap_content"
+                android:layout_below="@+id/notification_title"
+                android:layout_height="wrap_content"
+                android:layout_alignParentLeft="true"
+                android:layout_alignParentStart="true" />
+
+        </RelativeLayout>
+
+        <Chronometer
+            android:id="@+id/timer_text"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_alignParentRight="true"
+            android:layout_centerVertical="true"
+            android:textSize="20sp"
+            android:textAlignment="center"/>
+    </RelativeLayout>
+
+    <ImageView
+        android:id="@+id/expanded_template_image"
+        android:layout_width="match_parent"
+        android:layout_height="175dp"
```

Comment:
add this to dimens.xml
Also basic layout expanded height is 200dp. This one is 175dp. Any reason for making this smaller?